### PR TITLE
add alert when wallet connect is disconnect. Fix #536

### LINKED ISSUE #536: Wallet Connect: if error occurs on dapp side, RWallet WC page should alert user and then exit.
Currently if we use Mainnet to connect Sovryn Testnet, Sovryn will pop up an error saying "Please connect to RSK testnet", and RWallet Wallet Connect page will disappear. In situation like that, RWallet need to pop up a modal to notify user that the connection is closed by the dapp.

### PR DESCRIPTION
**Pull request by AmazingBeerBelly**
Add disconnect alert when WalletConnect disconnect.

CleanShot 2020-10-12 at 22 34 25@2x

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -155,6 +155,9 @@
         "leaveToDisconnect": "Sair para desconectar?",
         "leaveToDisconnectDesc": "Sair da página Wallet Connect irá fazer com que a RWallet se desconecte do Dapp atualmente conectado.",
         "leaveToDisconnectConfirm": "Você tem certeza que deseja desconectar?",
+        "disconnectAlertTitle": "A conexão de carteira está desconectada",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:158]:
```suggestion
        "disconnectAlertTitle": "Wallet Connect desconectada",
```

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -155,6 +155,9 @@
...
+        "disconnectAlertContent": "A conexão Wallet Connect com o Dapp \"%{dappName}\" está fechada.Isso pode ser devido ao fraco acesso à rede, interrupção do servidor Wallet Connect, ou rejeição unilateral Dapp.Ao clicar no botão embaixo você será navegado para o painel de carteira.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:159]:
```suggestion
        "disconnectAlertContent": "O Dapp \"%{dappName}\" foi desconectado da Wallet Connect. Pode ter ocorrido instabilidade no acesso à rede, interrupção do servidor da Wallet Connect, ou rejeição por parte do Dapp. Ao clicar no botão abaixo você será direcionado para a carteira.",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor changes Portuguese Brazilian

**Comment by bgorlick** [comment]
Some grammatical improvements to the message would be to use the below message:

The Wallet Connect connection to the Dapp "WalletConnect Example" is closed. This may be because of weak network access, Wallet Connect server interruptions, or a Dapp unilateral rejection (the Dapp itself rejected you directly for some reason). By clicking the button below you will be returned to the wallet dashboard.

**Comment by chrisli30** [comment]
> The Wallet Connect connection to the Dapp "WalletConnect Example" is closed. This may be because of weak network access, Wallet Connect server interruptions, or a Dapp unilateral rejection (the Dapp itself rejected you directly for some reason). By clicking the button below you will be returned to the wallet dashboard.

I agree. Let's remove the parens cause it looks informal. I also removed "Wallet Connect server interruptions". @AmazingBeerBelly please use below one.

The Wallet Connect connection to the Dapp "WalletConnect Example" is closed. This may be because of weak network access, or the Dapp rejected the wallet directly for some reason. 

By clicking the button below you will be returned to the wallet dashboard.

**Comment by bitcoinuser** [comment]
> @bitcoinuser could you double check the translation for Portuguese? Thanks!

Done.

**Comment by AmazingBeerBelly** [comment]
CleanShot 2020-10-13 at 15 03 35@2x

**Review by chrisli30 — APPROVED** [review]
Looks good to me. Great work guys!

**Review by imstar15 — APPROVED** [review]

**Pull request merged**